modules/mbedtls/zephyr_time.c

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mbedtls/platform_time.h>
+#include <zephyr/kernel.h>
+
+mbedtls_ms_time_t mbedtls_ms_time(void)
+{
+	return (mbedtls_ms_time_t)k_uptime_get();
+}
```

Comment:
Invoking the C compiler for just this little function slows down the build.

Let's move it into zephyr_init.c.

zephyr_init.c already contains Zephyr-specific mbedtls platform implementations like mbedtls_hardware_poll.

Reply:
moved it